internal/analytics/analytics.go

```diff
@@ -15,6 +16,16 @@ type Dispatcher interface {
 	Close()
 }
 
+var AnalyticsURL string
```

Comment:
This should still be recorded on the PixelReporter. Unlike with the API host the use of analytics is not pervasive, it's all a single instance.

Reply:
I've moved this to the pixel reporter but as stated above I can't confirm that there will be a single instance